Source/NSubstitute.Acceptance.Specs/MultipleThreads.cs

```diff
@@ -112,6 +112,35 @@ public void Create_Delegate_Substitute_From_Many_Threads()
             Task.AwaitAll(tasks);
         }
 
+        [Test]
```

Comment:
>It isn't a very good practice to have have multiple different assertions within the single test - it's harder to read them.

I would say that it is not a good practice to have multiple different assertions within a single test when the additional assertions make the test harder to read it.